schema.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE animals (
     escape_attempts     INT,
     neutered            BOOLEAN,
     weight_kg           DECIMAL,
+    species VARCHAR(250),
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires adding a species column to the animals table by using `ALTER TABLE` command, not by changing the previous create table query. Kindly do that.

Reply:
Fixed 🚀

Reply:
This issue is well resolved! ✔️